MSAL/test/unit/MSALPublicClientApplicationTests.m

```diff
@@ -132,63 +131,11 @@ - (void)testInit
     XCTAssertNotNil(application);
     XCTAssertNil(error);
     XCTAssertEqualObjects(application.clientId, UNIT_TEST_CLIENT_ID);
-    XCTAssertEqualObjects(application.redirectUri.absoluteString, UNIT_TEST_DEFAULT_REDIRECT_URI);
+    XCTAssertEqualObjects(application.redirectUri, UNIT_TEST_DEFAULT_REDIRECT_URI);
     
     application = nil;
 }
 
-- (void)testHandleMSALResponse_whenInvalid_returnsNo
```

Comment:
why this test got removed? we still have this public method?

Reply:
The method itself relies 100% on the common core. I will add to the msidcommon tests.